src/main/modules/oidc/index.ts

```diff
@@ -47,6 +48,8 @@ export class OidcMiddleware {
         if (req.session?.user) {
           res.locals.isLoggedIn = true;
           req.locals.api = getCaseApi(req.session.user, req.locals.logger);
+          req.locals.C100Api = caseApi(req.session.user, req.locals.logger);
```

Comment:
will this call affect for non C100 flow?

Reply:
No, this wont affect. This is done specifically for c100 rebuild app to consume the api by creating a singleton instance of the api class.